### DISCUSSION TITLE
How I can add a link in Laravel-Lang string ?

### DISCUSSION THREAD

**Discussion by Van4kk** [Laravel-Lang/common#21] [Q&A] [answered]
I have to do an translatable text which contains links. How can I do that, do I need to translate link and text separated or there is a way to do this ?
```
"link 1": "translated link 1",
"link 2" : "translated link 2",
"text up to link 1": "translated  text up to link 1",
"text between links": "translated text between links",
"text after link 2" : "translated text after link 2",
```

**Comment by andrey-helldar** [comment, marked as answer]
You can create your own repository using our template: https://github.com/Laravel-Lang/translations-template

In the source folder, delete all unnecessary files, create your own, for example, `links.json`.

Then run the `vendor/bin/lang create en` command in the console.

The localization code can be any available from the list: https://github.com/Laravel-Lang/publisher/blob/main/src/Constants/Locales.php

After that, change the files in the [`src`](https://github.com/Laravel-Lang/translations-template/tree/main/src) folder for yourself and publish the package.

Next, using our [`publisher`](https://publisher.laravel-lang.com/installation/), your translations will be automatically installed in the application when you call the `php artisan lang:update` command.

Simple steps: https://github.com/Laravel-Lang/translations-template#steps

**Reply by Van4kk** [reply]
Thank you !

**Reply by andrey-helldar** [reply]
For example: https://github.com/Laravel-Lang/http-statuses
